src/components/text-editor/prosemirror-adapter/plugins/image/inserter.ts

```
@@ -261,19 +283,253 @@
+ */
+const processImageSource = (view: EditorView, src: string): void => {
+    const sourceType = detectImageSourceType(src);
+
+    switch (sourceType) {
+        case 'data-url':
+            processDataUrlImage(view, src);
+            break;
+        case 'external-url':
+            processExternalUrlImage(view, src);
+            break;
+        case 'unknown':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)